src/eval.rs

```diff
@@ -18,10 +18,11 @@ use crate::{
 #[derive(Clone)]
 pub struct MiriConfig {
     pub validate: bool,
+    pub communicate: bool,
```

Comment:
Please add a doc comment.

Reply:
Bonus points if you also doc-comment the undocumented existing fields. :) But that's optional.